### ISSUE TITLE
Payment method credit cart doesn't show additionalDescription any more

### ISSUE THREAD

**Issue by luckyduck** [PAYONE-GmbH/shopware-5#390]
Our team is working on a shopware 5 project which relies on payone to accept credit card payments. After an update from probably 4.2.2 to 4.3.0, the additional description including the logo is now only visible when you select credit card as payment method and then VISA as card explicitly. To see how the checkout looks now, see now.png.

![now](https://user-images.githubusercontent.com/400837/94240219-47115080-ff13-11ea-90ce-38ad07bf469b.png)

In the current setup, you have to select the payment method. In the past, the checkout showed the payment logo(s) without this step. Kinda like when you explicitly select a payment method now:

![after](https://user-images.githubusercontent.com/400837/94240468-a1121600-ff13-11ea-8447-95ff3335dc5c.png)

But: the additionalDescription is already available in the HTML. It was hidden by JavaScript (seems like this is happening in mopt_payment.js), changeAdditionalDescription. If I remove display: none, it is visible just like before.

Do you have any pointers? Can you help?

**Comment by hreinberger** [comment]
Hi @luckyduck sorry for the late reply. This looks like our automatic credit card type detection got switched off somehow.
![grafik](https://user-images.githubusercontent.com/40197225/95175212-7188c680-07bb-11eb-98bc-1312ffb105a4.png)
could you try switching it on in the PAYONE credit card configuration? Please re-compile the theme after switching the option to on.

**Comment by luckyduck** [comment]
Hi. It is in fact off. But if I turn it on, it is still only visible after I enter a Credit Card Number and I am not able to choose between cards manually. It still doesnt show the logo without having selected "Kreditkarte" (as shown in the screenshot before)

**Comment by hreinberger** [comment]
Sorry for the late reply, I seem to have missed the notification.
I also think I finally got your actual issue. I've setup two shops for reproduction. One with Version 4.2.0 and one with 4.4.0 and I found the same behavior:
In 4.2.0, html in the additional description of the payone visa payment method showed up, even when the payment method wasn't expanded:
![grafik](https://user-images.githubusercontent.com/40197225/99688504-2c2c0a00-2a86-11eb-9fc6-7b36675fbccc.png)
results in
![grafik](https://user-images.githubusercontent.com/40197225/99687969-8a0c2200-2a85-11eb-8c50-0ea10dec21f1.png)
However in 4.4.0
![grafik](https://user-images.githubusercontent.com/40197225/99688040-a14b0f80-2a85-11eb-9f23-9f1e4af1bb45.png)
results in
![grafik](https://user-images.githubusercontent.com/40197225/99688144-bfb10b00-2a85-11eb-87ac-bad732668a97.png)

I agree, there should be a possibility to show additional text or html when the payment methods aren't expanded. I'll check internally what commit was responsible and how we can revert the behavior to the pre-4.4.0 one.
